src/services/jwt.service.js

```diff
@@ -0,0 +1,52 @@
+import jwt from 'jsonwebtoken';
+
+const generateAccessToken = (user) => {
+  return jwt.sign(user, process.env.JWT_KEY, {
+    expiresIn: '10m',
+  });
+};
+
+const generateRefreshToken = (user) => {
+  return jwt.sign(user, process.env.JWT_REFRESH_KEY, {
+    expiresIn: '10m',
```

Comment:
The refresh token expiration time is also set to '10m'. Refresh tokens usually have a longer lifespan than access tokens. Consider extending this duration based on your application's needs.